static/css/v3/foundations.css

```
@@ -164,12 +180,19 @@ html.v3 code {
 
 /* Container for v3 content */
 .v3-container {
+  --v3-shell-gutter: var(--space-large);
   width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)